controllers/depresolver/depresolver.go

```diff
@@ -60,17 +60,19 @@ func (f LogFormat) String() string {
 }
 
 // EdgeDNSType specifies to which edge DNS is k8gb connecting
-type EdgeDNSType int
+type EdgeDNSType string
 
 const (
 	// DNSTypeNoEdgeDNS is default DNSType. Is used during integration testing when no edgeDNS provider exists
-	DNSTypeNoEdgeDNS EdgeDNSType = 1 << iota
+	DNSTypeNoEdgeDNS EdgeDNSType = "EmptyDNS"
 	// DNSTypeInfoblox type
-	DNSTypeInfoblox
+	DNSTypeInfoblox EdgeDNSType = "Infoblox"
 	// DNSTypeRoute53 type
-	DNSTypeRoute53
+	DNSTypeRoute53 EdgeDNSType = "Route53"
 	// DNSTypeNS1 type
-	DNSTypeNS1
+	DNSTypeNS1 EdgeDNSType = "NS1"
+	// DNSTypeMultipleProviders type
+	DNSTypeMultipleProviders EdgeDNSType = "DNSError"
```

Comment:
Same here, it's good to reflect const name in value:
```suggestion
	DNSTypeMultipleProviders EdgeDNSType = "MultipleProviders"
```